rpc/common_test.go

```diff
@@ -241,7 +241,7 @@ func TestGetPluginCapabilities(t *testing.T) {
 			defer os.RemoveAll(tmp)
 			identity := &fakeIdentityServer{
 				getPluginCapabilitiesResponse: test.output,
-				err: injectedErr,
+				err:                           injectedErr,
```

Comment:
I'm not sure why this passed testing in https://github.com/kubernetes-csi/csi-lib-utils/pull/20.

Reply:
The explanation is that the Docker image that is currently used by the mandatory pre-merge check l only has Go 1.10.1. We'll transition to current Go as part of replacing the Prow check.